src/ui/page/profile/view.html

```diff
@@ -65,7 +65,7 @@ <h2 class="heading heading-medium ">
                         <tr>
                             <td class="key-value-list__key key-value-list__key--auto">{{i18n('profile_department_label')}}</td>
                             <td class="key-value-list__value key-value-list__auto--auto lpg-department">
-                                {{signedInUser.department === 'other' ? 'Other' : i18n('departments')[signedInUser.department] }}
+                                {{signedInUser.department === ('other' || 'select') ? 'Other' : i18n('departments')[signedInUser.department] }}
```

Comment:
Did you mean `signedInUser.department === 'other' || signedInUser.department === 'select'` ?